Mango API/src/com/infiniteautomation/mango/rest/latest/ServerRestController.java

```diff
@@ -291,6 +291,7 @@ public ResponseEntity<List<SessionInformation>> listSessions(@AuthenticationPrin
         @ApiResponse(code = 500, message = "Internal error", response = ResponseEntity.class),})
     @RequestMapping(method = {RequestMethod.GET}, value = "system-info")
     public ResponseEntity<Map<String, Object>> getSystemInfo(@AuthenticationPrincipal PermissionHolder user) {
+        System.out.println("Testing on the fly thing");
```

Comment:
Revert all changes to this class.  When done we will squash this into a single commit to remove this change entirely.

Reply:
changes removed.